cosmo_tester/framework/fixtures.py

```diff
@@ -10,12 +10,14 @@ def image_based_manager(
     """Creates a cloudify manager from an image in rackspace OpenStack."""
     hosts = TestHosts(
             cfy, ssh_key, module_tmpdir, attributes, logger)
+    err_raised = True
     try:
         hosts.create()
         hosts.instances[0].use()
         yield hosts.instances[0]
+        err_raised = False
     finally:
-        hosts.destroy()
+        hosts.destroy(err_raised)
```

Comment:
Why not add a new function here that runs the "self._save_manager_logs(err_raised)"?
It is much cleaner because destroy doest suggest save logs.

Reply:
@tyacbovi Because in other test/s I see that this code has been copied and therefore the common point is within the function [here for example](https://github.com/cloudify-cosmo/cloudify-system-tests/blob/CY-586/cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py#L66).
Having said that, I'll need to add this exception raised catch to all of them. Any other ideas?